azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDK.kt

```diff
@@ -46,6 +46,7 @@ internal interface CallingSDK {
     fun getIsRecordingSharedFlow(): SharedFlow<Boolean>
     fun getIsMutedSharedFlow(): SharedFlow<Boolean>
     fun getCallingStateWrapperSharedFlow(): SharedFlow<CallingStateWrapper>
+    fun getCallIdSharedFlow(): SharedFlow<String?>
```

Comment:
@pavelprystinka it can be a state flow? as id will be unique

Reply:
Updated